operationsbus/handlers.go

```diff
@@ -221,15 +236,19 @@ func NewOperationContainerHandler(errHandler ErrorHandlerFunc, operationContaine
 			case *NonRetryError:
 				// Cancel the operation
 				logger.Info("OperationContainerHandler: Setting operation as Cancelled.")
-				// err = operationContainerOperationCancel(ctx, body.OperationId)
 				updateOperationStatusRequest = &oc.UpdateOperationStatusRequest{
 					OperationId: body.OperationId,
 					Status:      oc.Status_CANCELLED,
```

Comment:
If this is a situation where we give up entirely, then this should be Failed. Cancelled is a user-initiated operation

Reply:
Ah, you're correct. Changed to Failed.